cmd/keeper/cmd/keeper.go

```diff
@@ -114,6 +114,9 @@ type config struct {
 	pgSUPasswordFile        string
 	pgInitialSUUsername     string
 	pgInitialSUPasswordFile string
+
+	neverMaster             bool
+	neverSynchronousReplica bool
```

Comment:
Just a poll... Since I don't lilke negated flags. What about `canBeMaster` `canBeSynchronousReplica` that defaults to true?

Reply:
I think you're right about that, negative flags aren't usually the most user friendly. We originally chose this as it felt natural to specify that you couldn't become a particular role, but I don't think that makes sense on second thought. 

Let's go with your suggestion!